x-pack/plugins/security_solution/public/common/containers/sourcerer/index.test.tsx

```diff
@@ -411,6 +416,58 @@ describe('Sourcerer Hooks', () => {
         ]);
       });
     });
+
+    it('Should put any excludes in the index pattern at the end of the pattern list', async () => {
+      await act(async () => {
+        store = createStore(
+          {
+            ...mockGlobalState,
+            sourcerer: {
+              ...mockGlobalState.sourcerer,
+              sourcererScopes: {
+                ...mockGlobalState.sourcerer.sourcererScopes,
+                [SourcererScopeName.default]: {
+                  ...mockGlobalState.sourcerer.sourcererScopes[SourcererScopeName.default],
+                  selectedPatterns: [
+                    'endgame-*',
+                    'auditbeat-*',
+                    'filebeat-*',
+                    'winlogbeat-*',
+                    '-filebeat-*',
+                    'packetbeat-*',
+                    '-packetbeat-*',
+                    'traces-apm*',
+                    'apm-*-transaction*',
```

Comment:
This test already (implicitly) verifies the `sortWithExcludesAtEnd` function sorts all of the _includes_ entries, because entries like `endgame-*` appear before `auditbeat-*` in the input.

Consider moving the `-packetbeat-*` exclude pattern to the top, per the suggestion below:

```ts
                    '-packetbeat-*',
                    'endgame-*',
                    'auditbeat-*',
                    'filebeat-*',
                    'winlogbeat-*',
                    '-filebeat-*',
                    'packetbeat-*',
                    'traces-apm*',
                    'apm-*-transaction*',
```

because that will also test that the _excludes_ pattern is always sorted.